server/service/integration_mdm_test.go

```diff
@@ -12595,114 +12626,114 @@ func (s *integrationMDMTestSuite) TestAppleDDMBatchUpload() {
+	// 	{Name: "N2", Contents: decls[0]},
+	// }}, http.StatusUnprocessableEntity)
+	// errMsg = extractServerErrorText(res.Body)
+	// require.Contains(t, errMsg, "A declaration profile with this identifier already exists.")
+
+	// // Create 2 declarations
+	// s.Do("POST", "/api/latest/fleet/mdm/profiles/batch", batchSetMDMProfilesRequest{Profiles: []fleet.MDMProfileBatchPayload{
+	// 	{Name: "N1", Contents: decls[0]},
+	// 	{Name: "N2", Contents: decls[1]},
+	// }}, http.StatusNoContent)
+
+	// var resp listMDMConfigProfilesResponse
+	// s.DoJSON("GET", "/api/latest/fleet/mdm/profiles", &listMDMConfigProfilesRequest{}, http.StatusOK, &resp)
+
+	// require.Len(t, resp.Profiles, 2)
+	// require.Equal(t, "N1", resp.Profiles[0].Name)
+	// require.Equal(t, "darwin", resp.Profiles[0].Platform)
+	// require.Equal(t, "N2", resp.Profiles[1].Name)
+	// require.Equal(t, "darwin", resp.Profiles[1].Platform)
+
+	// // Create 2 new declarations. These should take the place of the first two.
+	// s.Do("POST", "/api/latest/fleet/mdm/profiles/batch", batchSetMDMProfilesRequest{Profiles: []fleet.MDMProfileBatchPayload{
+	// 	{Name: "N3", Contents: decls[2]},
+	// 	{Name: "N4", Contents: decls[3]},
+	// }}, http.StatusNoContent)
+
+	// s.DoJSON("GET", "/api/latest/fleet/mdm/profiles", &listMDMConfigProfilesRequest{}, http.StatusOK, &resp)
+
+	// require.Len(t, resp.Profiles, 2)
+	// require.Equal(t, "N3", resp.Profiles[0].Name)
+	// require.Equal(t, "darwin", resp.Profiles[0].Platform)
+	// require.Equal(t, "N4", resp.Profiles[1].Name)
+	// require.Equal(t, "darwin", resp.Profiles[1].Platform)
+
+	// // replace only 1 declaration, the other one should be the same
+
+	// s.Do("POST", "/api/latest/fleet/mdm/profiles/batch", batchSetMDMProfilesRequest{Profiles: []fleet.MDMProfileBatchPayload{
+	// 	{Name: "N3", Contents: decls[2]},
+	// 	{Name: "N5", Contents: decls[4]},
+	// }}, http.StatusNoContent)
+
+	// s.DoJSON("GET", "/api/latest/fleet/mdm/profiles", &listMDMConfigProfilesRequest{}, http.StatusOK, &resp)
+
+	// require.Len(t, resp.Profiles, 2)
+	// require.Equal(t, "N3", resp.Profiles[0].Name)
+	// require.Equal(t, "darwin", resp.Profiles[0].Platform)
+	// require.Equal(t, "N5", resp.Profiles[1].Name)
+	// require.Equal(t, "darwin", resp.Profiles[1].Platform)
+
+	// // update the declarations
+
+	// s.Do("POST", "/api/latest/fleet/mdm/profiles/batch", batchSetMDMProfilesRequest{Profiles: []fleet.MDMProfileBatchPayload{
+	// 	{Name: "N3", Contents: newDeclBytes(2, `"foo": "bar"`)},
+	// 	{Name: "N5", Contents: newDeclBytes(4, `"bing": "baz"`)},
+	// }}, http.StatusNoContent)
+
+	// s.DoJSON("GET", "/api/latest/fleet/mdm/profiles", &listMDMConfigProfilesRequest{}, http.StatusOK, &resp)
+
+	// require.Len(t, resp.Profiles, 2)
+	// require.Equal(t, "N3", resp.Profiles[0].Name)
+	// require.Equal(t, "darwin", resp.Profiles[0].Platform)
+	// require.Equal(t, "N5", resp.Profiles[1].Name)
+	// require.Equal(t, "darwin", resp.Profiles[1].Platform)
```

Comment:
why all of these are commented out?

Reply:
Thanks for catching that! I commented those out while debugging this test and forgot to restore them :)

Reply:
idk if this is a thing only in my browser, but it doesn't help that github doesn't highlight this file anymore. Maybe it's too large?

Reply:
I think you're right about the size impacting how GitHub renders the changes. It is the same for me---white text with no syntax formatting. Maybe we should split up the file at some point?